tsconfig.base.json

```diff
@@ -4,13 +4,13 @@
     "declaration": true,
     "declarationMap": true,
     "esModuleInterop": true,
-    "lib": ["ES2021"],
+    "lib": ["es2022"],
     "module": "NodeNext",
     "moduleResolution": "NodeNext",
     "resolveJsonModule": true,
     "skipLibCheck": true,
     "strict": true,
-    "target": "ES2021",
+    "target": "es2022",
```

Comment:
Two questions:

 1. Why did you change `lib` and `target` configs?
 1. Why `ES2022` and not `ES2023`?

Reply:
I wasn't aware of this Error `cause`; it seems pretty [compatible](https://developer.mozilla.org/en-US/docs/Web/JavaScript/Reference/Global_Objects/Error/cause#browser_compatibility) already. TIL.

I was just curious about the  `ES20**` config. I don't have a saying about it as I have no in-depth knowledge about which would be better or why, and neither if it could break something.

Reply:
According to these, it seems ok to target `ES2023`:
 - https://node.green/
 - https://kangax.github.io/compat-table/es2016plus/

Happy new year @camsjams 🎉